src/sentry/integrations/slack/unfurl/issues.py

```diff
@@ -63,10 +65,16 @@ def unfurl_issues(
     return out
 
 
+issue_link_regex = re.compile(
+    r"^https?\://(?#url_prefix)[^/]+/(?#organizations)[^/]+/(?#organization_slug)[^/]+/issues/(?P<issue_id>\d+)(?:/events/(?P<event_id>\w+))?"
+)
+
+customer_domain_issue_link_regex = re.compile(
+    r"^https?\://(?#url_prefix)[^/]+/issues/(?P<issue_id>\d+)(?:/events/(?P<event_id>\w+))?"
```

Comment:
```suggestion
    r"^https?\://(?#url_prefix)[^.]+/issues/(?P<issue_id>\d+)(?:/events/(?P<event_id>\w+))?"
```

You'll hit a `.` before you hit a `/` here too.

Reply:
This suggestion is not applicable here since we're not matching on an org slug subdomain.